### ISSUE TITLE
0.233.1 memory leak with error QueryFailedError: SQLITE_ERROR: table execution_entity has no column named data

### ISSUE THREAD

**Issue by Pierric82** [n8n-io/n8n#6608]
**Describe the bug**
Since yesterday n8n running on Docker is logging errors continuously:
> QueryFailedError: SQLITE_ERROR: table execution_entity has no column named data

In addition, its memory usage grows very fast, to the point of crashing my system after some time.
I use WatchTower to auto-upgrade containers and it looks like the container was upgraded yesterday, matching the beginning of the issues. The currently running version is 0.233.1.
In appearance though, n8n runs fine, I can access the UI, open my automations, etc.
I haven't found any particular instructions to run database migrations that could be linked with the error message.

**To Reproduce**
Run the docker container `n8nio/n8n:latest` on an x86-amd64 architecture (Docker is itself running inside a Proxmox VM running Debian 11) using a pre-existing `/home/node/.n8n` folder mount.

**Expected behavior**
Things running fine as usual, with no memory leak, no errors in the logs.

**Environment (please complete the following information):**

- OS: Docker on Debian 11
- n8n Version 0.233.1
- Node.js Version as packaged in the Docker container
- Database system SQLite
- Operation mode probably 'own' as I don't know what this refers to

**Additional context**

**Comment by Joffcom** [comment]
Hey @Pierric82 

This is an issue that was caused by use a couple of days ago when we pulled a released that had the latest tag.

Your instance would have been upgraded to 0.234.0 which had a one way database migration, it would then have downgraded to 0.233.1 when we changed the tag because of a migration issue for MySQL databases.

The 0.234.1 release from yesterday will fix this and should be marked as latest so in theory watchtower should already have picked it up.

If it hasn't doing an update to 0.234.1 manually should sort this out.
